webviz_subsurface/plugins/_inplace_volumes.py

```diff
@@ -20,7 +20,7 @@
 from .._abbreviations.number_formatting import table_statistics_base
 
 
-class InplaceVolumes(WebvizPluginABC):
+class InplaceVolumesOld(WebvizPluginABC):
```

Comment:
Something to consider (not sure what is best):

Unless the new `InplaceVolumes` is a drop-in-replacement of the old one (i.e. takes the same argument names), you can also consider deprecating the existing plugin (`webviz-config` recently got functionality for this: https://github.com/equinor/webviz-config/blob/master/CONTRIBUTING.md#deprecate-plugins-or-arguments), and then try (if possible) to find a new name for the new plugin.

Benefit: Existing webviz configurations/setups will continue to exist, and users are notified that they should use something else. Downside: Need a new name for the new plugin, and the transition might take a bit longer (no there is no hard error :laughing: ).

Reply:
Yes, this was the quickest solution for me to get the new plugin running locally while working 😉 I agree I like the new deprecation system, and we can come up with a new clever name for this one!